### PR TITLE
Chinese translation

### PR DESCRIPTION
**Pull request by dingjs**
Got the tranlsation reviewed and fix some typos.
## Description
 fix some typos.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added unit tests to cover my changes.
- [ ] I have added/updated functional tests in [mint](https://github.com/minio/mint). (If yes, add `mint` PR # here: )
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by harshavardhana — APPROVED** [review]
LGTM

File: docs/zh_CN/shared-backend/README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Minio共享模式是为了解决在真实场景中存在的一些问题，而且
 
 ## 2. 在共享后端存储上运行Minio
 
-为了让共享后端存储的Minio实例运行起来，你需要启动多个Minio服务，这些服务只想同一个后端存储。下面我们就来讲解如何设置。
+为了让共享后端存储的Minio实例运行起来，你需要启动多个Minio服务，这些服务指向同一个后端存储。下面我们就来讲解如何设置。
```
Comment (yihuaf) [inline docs/zh_CN/shared-backend/README.md]: 为了让共享后端存储的Minio实例运行起来 ->  为了让Minio在共享后端上运行起来

**Review by yihuaf — APPROVED** [review]
Looks good to me. Please address the one comment and check in.

File: docs/zh_CN/shared-backend/README.md (hunk 1 above)
Comment (deekoder) [reply docs/zh_CN/shared-backend/README.md]: @dingjs I will wait for you to address @yihuaf  comment above and then merge this PR.

**Review by deekoder — CHANGES_REQUESTED** [review]
PTAL @yihuaf  comments

File: docs/zh_CN/shared-backend/README.md (hunk 1 above)
Comment (dingjs) [reply docs/zh_CN/shared-backend/README.md]: @deekoder OK,I just fixed the issue.
@yihuaf 多谢，已修改。

**Comment by yihuaf** [comment]
@deekoder The latest change (commit: 7804a83) fixed my comment.

**Review by deekoder — APPROVED** [review]

**Pull request merged**